### ISSUE TITLE
Tags/Lists

### ISSUE THREAD

**Issue by dbarwikowski** [microsoft/winget-cli#2438]
### Description of the new feature / enhancement

When I install new software I would like to tag it / add it to a list so that when I export json file I can choose if I want full setup or just items that are tagged / on a list

### Proposed technical implementation details

`winget install Microsoft.WindowsTerminal --tag my-tag`
or
`winget install Microsoft.WindowsTerminal --add-to-list my-list`

**Comment by denelon** [comment]
We've recently been discussing mechanisms for excluding packages from `winget upgrade --all`.
* https://github.com/microsoft/winget-cli/issues/2339

One of the thoughts has been to also exclude those from `winget export packages.json`.

Would something like that express what you are asking for?

**Comment by Masamune3210** [comment]
I at least would love it to be optional whether it is omitted from export or not. Usually, I just want to keep an app from being upgraded, but I use export to back up apps so I can install them on another computer or on a new installation say after a fresh install of Windows.  Just my two cents

**Comment by denelon** [comment]
@Massamune3210 I could see a `winget export packages.json --include-excluded`

**Comment by Masamune3210** [comment]
that would be perfect, at least for me, though I don't see any way someone else could complain about it

**Comment by dbarwikowski** [comment]
> One of the thoughts has been to also exclude those from `winget export packages.json`.
> 
> Would something like that express what you are asking for?

Yes, it will do what I have in mind, but it will also make it difficult.
I will have to exclude every default program to achieve what I want.
That looks like a lot of work. 

Maybe that is something that would make it easier:
`winget exclude --all`
